vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/storage/AssertingWebElementsStorage.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd.steps.ui.web.storage;
+
+import java.util.List;
+
+import com.google.common.eventbus.Subscribe;
+
+import org.openqa.selenium.WebElement;
+import org.vividus.softassert.event.AssertionEvent;
+import org.vividus.testcontext.TestContext;
+
+public class AssertingWebElementsStorage implements IAssertingWebElementsStorage
```

Comment:
why can't it be a part of `..UiContext`?

Reply:
discussed

Reply:
Discussed that the WebUiContext is responsible for saving elements within its context and there is no need for separate component